### ISSUE TITLE
deployment failed - cannot unmarshal number into Go struct

### ISSUE THREAD

**Issue by czarny94** [conjure-up/conjure-up#1574]
- [NO] Is this problem already documented at https://docs.ubuntu.com/conjure-up/en/troubleshoot#common-spell-problems
- [NO] Is conjure-up running inside a virtual machine?
- [YES] Is this problem reproducible with `sudo snap refresh conjure-up --edge`?

## Please provide the output of the following commands

which juju 
/snap/bin/juju
juju version
2.5.0-bionic-amd64

which conjure-up
/snap/bin/conjure-up
conjure-up --version
conjure-up 2.6.4

lxc - not using lxc - only docker

cat /etc/lsb-release
DISTRIB_ID=Ubuntu
DISTRIB_RELEASE=18.04
DISTRIB_CODENAME=bionic
DISTRIB_DESCRIPTION="Ubuntu 18.04.1 LTS"

Please attach tarball of **~/.cache/conjure-up**:
[conjure-up.tar.gz](https://github.com/conjure-up/conjure-up/files/2791879/conjure-up.tar.gz)


## What Spell was Selected?
Canonical distribution of Kubernetes, Core Kubernetes

## What provider (aws, maas, localhost, etc)?
Vsphere

## Commands ran

Please outline what commands were run to install and execute conjure-up:
sudo snap install conjure-up --classic
sudo snap install juju --classic
conjure-up
## Additional Information
```
Reason:                                                                                                                                                                            
json: cannot unmarshal number into Go struct field ApplicationDeploy.devices of type devices.Constraints                                                                 
                                                                                                                                                                                              
Review log messages at /home/warlock/.cache/conjure-up/conjure-up.log If appropriate, please submit a bug here: https://github.com/conjure-up/conjure-up/issues/new  `
[conjure-up.log](https://github.com/conjure-up/conjure-up/files/2791878/conjure-up.log)
```
The error occurs immediately after the creation of the first machine on the vSphere, it occurs shortly after the deployment window appears.

**Comment by czarny94** [comment]
Problem solved after downgrading juju from 2.5.0 to 2.4.7

**Comment by JohnnyElvis** [comment]
I have the same issue with juju 2.6.3-bionic-amd64, conjure-up version: 2.6.8

Can you provide necessary steps to to downgrade juju?
